tools/docker/besu-all-in-one/Dockerfile_v21.1.x

```diff
@@ -0,0 +1,30 @@
+ARG BESU_VERSION=21.1.6
```

Comment:
@AzaharaC I know this file naming pattern has been precedent in the Fabric AIO image, but it is something that I wanted to steer us away from because VSCode doesn't recognize Dockerfile's with custom names (syntax helper).
Could you refactor this in a way that instead of the file being `Dockerfile_v21.1.x` there is a directory called `v21_1_x` and then three's a `Dockerfile` in there with the contents of this file that I'm commenting on. Sorry for the nit picking as usual.

Reply:
Don't worry Peter, I have already changed this.